features/facebookImport/src/model/importers/utils/importer-util.js

```diff
@@ -21,17 +21,22 @@ async function readJSONFile(relativeFilePath, zipFile) {
     return readFullPathJSONFile(entry);
 }
 
+function decode(s) {
+    let d = new TextDecoder();
+    let a = s.split("").map((r) => r.charCodeAt());
+    return d.decode(new Uint8Array(a));
+}
+
 async function readFullPathJSONFile(entry) {
     const rawContent = await entry.getContent();
-    const fileContent = new TextDecoder("utf-8").decode(rawContent);
-
+    const fileContent = decode(new TextDecoder().decode(rawContent));
     if (!fileContent) {
         throw new MissingContentImportException(entry._id);
     }
 
     return JSON.parse(fileContent, (key, value) => {
         if (typeof value === "string") {
-            return decodeURIComponent(escape(value));
+            return decodeURIComponent(value);
```

Comment:
Interesting, so `escape` would (kind of) URL encode stuff, and then we would URL decode it right after... Was this some kind of sanitation measure?

Now we stopped it with the `escape` - can `value` still be URL encoded?